From: Yunqiang Su <wzssyqa@gmail.com>
Subject: cmake - FindJNI add mips64(el) mipsn32(el) support
Date: Tue, 6 May 2014 16:06:45 +0800

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=747174

Add support for mips64 mips64el mipsn32 mipsn32el in
Modules/FindJNI.cmake.

Index: cmake/Modules/FindJNI.cmake
===================================================================
--- cmake.orig/Modules/FindJNI.cmake
+++ cmake/Modules/FindJNI.cmake
@@ -53,7 +53,7 @@ macro(java_append_library_directories _v
     elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^mips")
         # mips* machines are bi-endian mostly so processor does not tell
         # endianess of the underlying system.
-        set(_java_libarch "${CMAKE_SYSTEM_PROCESSOR}" "mips" "mipsel" "mipseb")
+        set(_java_libarch "${CMAKE_SYSTEM_PROCESSOR}" "mips" "mipsel" "mipseb" "mips64" "mips64el" "mipsn32" "mipsn32el")
     elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(powerpc|ppc)64le")
         set(_java_libarch "ppc64" "ppc64le")
     elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(powerpc|ppc)64")
